File: 02_Estructuras de Control/02_Condicionales/02_Dobles/CSharp/ejemplo1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejemplo1Multiple
{
    internal class Program
    {
        static void Main(string[] args)
        {
            /* Programa para determirar si un número es impar y múltiplo de 5
             */

            Console.ForegroundColor = ConsoleColor.Yellow;
            Console.WriteLine("===== Cálculo de un número si es impar y múltiplo de 5 =====");
            Console.ResetColor();
            Console.WriteLine();

            // Entrada

            Console.Write("Ingrese un número: ");
            int numero = int.Parse(Console.ReadLine());

            // Proceso + Salida

            if (numero % 2 != 0 && numero % 5 == 0)
            {
                Console.ForegroundColor = ConsoleColor.Green;
                Console.WriteLine("El número es impar y múltiplo de 5.");
                Console.ResetColor();
            }
            else
            {
                Console.ForegroundColor = ConsoleColor.Red;
                Console.WriteLine("El número NO es impar y múltiplo de 5.");
                Console.ResetColor();
            }

            Console.WriteLine("\nPresione cualquier tecla para salir...");
            Console.ReadKey();
        }
    }
}

File: 02_Estructuras de Control/02_Condicionales/02_Dobles/Python/ejemplo1.py
# Programa para determinar si un número es impar y múltiplo de 5

print("===== Cálculo de un número si es impar y múltiplo de 5 =====\n")

# Entrada
numero = int(input("Ingrese un número: "))

# Proceso + Salida
if numero % 2 != 0 and numero % 5 == 0:
    print("El número es impar y múltiplo de 5.")
else:
    print("El número NO es impar y múltiplo de 5.")

input("\nPresione Enter para salir...")